<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1198</width>
    <height>783</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="Aunthentication" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1201</width>
     <height>801</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#FFF2CC;
</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <widget class="QTabWidget" name="Frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1201</width>
      <height>721</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#CAEFD1</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Register</string>
     </attribute>
     <widget class="QLineEdit" name="email_input">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#eeeedd</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 italic 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QComboBox" name="gender_input">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>140</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#eeeedd</string>
      </property>
      <item>
       <property name="text">
        <string>Male</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Female</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="member_input">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>200</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#eeeedd</string>
      </property>
      <item>
       <property name="text">
        <string>Member</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Non-Member</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 italic 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Id</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>180</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 italic 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Membership</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="name_input">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#eeeedd</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>60</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 italic 12pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>Contact</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_input">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#eeeedd</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>120</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 italic 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Gender</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="contact_input">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>80</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#eeeedd</string>
      </property>
     </widget>
     <widget class="QCalendarWidget" name="birthday_widget">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>280</y>
        <width>201</width>
        <height>190</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#555555</string>
      </property>
     </widget>
     <widget class="QCalendarWidget" name="expiry_input">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>201</width>
        <height>190</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#555555</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 italic 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="address_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>480</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 italic 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
     <widget class="QPushButton" name="register_button">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>580</y>
        <width>441</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#eeeedd
</string>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>20</y>
        <width>661</width>
        <height>621</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#555555</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="address_input">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>520</y>
        <width>441</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#eeeedd</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 italic 12pt &quot;Segoe UI&quot;;
</string>
      </property>
      <property name="text">
       <string>Register Member</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 italic 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Expiry</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>240</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 italic 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Birthday</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Edit</string>
     </attribute>
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>641</width>
        <height>641</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#555555</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>60</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 italic 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>20</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 italic 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Search Member</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="name_search_input">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>80</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#eeeedd</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>60</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 italic 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_search_input">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>80</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#eeeedd</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SearchButton">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>120</y>
        <width>481</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#eeeedd</string>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>220</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 italic 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Update The Selected Expiry</string>
      </property>
     </widget>
     <widget class="QCalendarWidget" name="Expiry_edit">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>260</y>
        <width>481</width>
        <height>190</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#555555</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Save_Date_Bt">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>480</y>
        <width>481</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#eeeedd</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Login</string>
     </attribute>
     <widget class="QLabel" name="name_label">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>140</y>
        <width>401</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 italic 18pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="expiry_label">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>220</y>
        <width>261</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 italic 18pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_entry">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>280</y>
        <width>261</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#eeeedd</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Login_bt">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>340</y>
        <width>261</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#eeeedd</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>80</y>
        <width>281</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 italic 20pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Members Login</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Login Data</string>
     </attribute>
     <widget class="QTableView" name="tableView_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>261</width>
        <height>561</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#555555;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 900 italic 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Member Login Data</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
